examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -105,9 +104,10 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
 		// to stamp it on something that would rather not know it (e.g. the model).
 		const attach = async () => {
 			await container.attach(this.generateCreateNewRequest());
-			const resolved = container.resolvedUrl;
-			ensureFluidResolvedUrl(resolved);
-			return resolved.id;
+			if (!container.resolvedUrl) {
```

Comment:
Prefer strict boolean expressions here and elsewhere.  Now that I look at it I'm shocked how many places we have the lint rule disabled

```suggestion
			if (container.resolvedUrl === undefined) {
```